KomaMRICore/src/simulation/GPUFunctions.jl

```diff
@@ -9,11 +9,16 @@ const use_cuda = Ref{Union{Nothing,Bool}}(nothing)
 Simple function to print the CUDA devices available in the host.
 """
 print_gpus() = begin
-	println( "$(length(devices())) CUDA capable device(s)." )
-	for (i,d) = enumerate(devices())
-		u = i == 1 ? "*" : " "
-		println( "  ($(i-1)$u) $(name(d))")
-	end
+    @info "Loading GPUs"
+    if has_cuda()
```

Comment:
I would do
```julia
check_use_cuda()
if use_cuda[]
```

Reply:
Edited in commit [9079e70](https://github.com/cncastillo/KomaMRI.jl/commit/9079e706da5dff875166b6ae0e8cc17f3872f68d).